_includes/global_fields.md

```diff
@@ -4,6 +4,7 @@ A set of common fields that appear on all resources is included first, for refer
 
 | Name          | Type      | Description
 |-----------    |-----------|-----------|--------------
+|id 			| string    | A string representing a local identifier for the resource on the local system.  It may be a string, integer, guid, etc.  This value MUST be suitable for use in a helper function such as add_tags, add_lists to identify the corresponding resource
```

Comment:
Yes, though depending on how you look at it, it's also a con. Take the Civi implementation. Their urls look like https://samplesystem.org/api/v1/tags.php?id=X

So we'd either have to mandate URL structures, or allow for some kind of discoverable templating, either of which has complications.